code/modules/admin/bluespacetech.dm

```diff
@@ -193,6 +193,15 @@ ADMIN_VERB_ADD(/client/proc/cmd_dev_bst, R_ADMIN|R_DEBUG, TRUE)
 	status_flags ^= GODMODE
 	src << span("notice", "God mode is now [status_flags & GODMODE ? "enabled" : "disabled"]")
 
+/mob/living/carbon/human/bst/verb/allow_unequip()
```

Comment:
I've been meaning to add this verb for a while, this PR gave me a reason to do it.

Allows BSTs to take off their items, useful when debugging worn equipment